libsolidity/ast/Types.cpp

```diff
@@ -3090,6 +3098,19 @@ string FunctionType::canonicalName() const
 	return "function";
 }
 
+string FunctionType::humanReadableName() const
+{
+	switch (m_kind)
+	{
+	case Kind::Error:
+		return "error " + m_declaration->name() + signatureParameterList(m_parameterTypes, true);
+	case Kind::Event:
+		return "event " + m_declaration->name() + signatureParameterList(m_parameterTypes, true);
+	default:
+		return Type::toString();
```

Comment:
Can you explicitly pass `false` to `toString()` and also add the parameter name?
```suggestion
	case Kind::Error:
		return "error " + m_declaration->name() + signatureParameterList(m_parameterTypes, /* _short */ true);
	case Kind::Event:
		return "event " + m_declaration->name() + signatureParameterList(m_parameterTypes, /* _short */ true);
	default:
		return Type::toString(/* _short */ false);
```

Otherwise it's not that easy to spot. I just noticed this and started wondering why we don't use the same value. So apparently `_short` affects only whether we print the location and nothing else. Might be good to add a comment about that because that's not obvious.

Reply:
Done.